assets/src/setup/pages/save/index.js

```diff
@@ -41,16 +103,88 @@ export function Save() {
 	}, [ didSaveDeveloperToolsOption, savingDeveloperToolsOption, saveDeveloperToolsOption ] );
 
 	if ( savingOptions || savingDeveloperToolsOption ) {
-		return <Loading />;
+		return <Saving />;
 	}
 
-	if ( ! didSaveOptions || ! didSaveDeveloperToolsOption ) {
-		return null;
+	let heading = __( 'Congratulations!', 'amp' );
+	if ( 'standard' === themeSupport ) {
+		heading = __( 'Your site is ready', 'amp' );
 	}
 
 	return (
-		<div>
-			{ __( 'Options saved', 'amp' ) }
+		<div className="done grid grid-5-4">
+			<div>
+				<h1>
+					{ heading }
+				</h1>
+				<p>
+					{ getDescription( themeSupport ) }
+				</p>
+			</div>
+			<div className="done__preview-container">
+				<p className="reader-summary__caption">
+					<svg width="13" height="20" viewBox="0 0 13 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+						<path d="M1.66211 0.265625H11.2246C11.5684 0.265625 11.8496 0.390625 12.0684 0.640625C12.3184 0.859375 12.4434 1.14063 12.4434 1.48438V18.2656C12.4434 18.6094 12.3184 18.9062 12.0684 19.1562C11.8496 19.375 11.5684 19.4844 11.2246 19.4844H1.66211C1.31836 19.4844 1.02148 19.375 0.771484 19.1562C0.552734 18.9062 0.443359 18.6094 0.443359 18.2656V1.48438C0.443359 1.14063 0.552734 0.859375 0.771484 0.640625C1.02148 0.390625 1.31836 0.265625 1.66211 0.265625ZM10.0527 14.6562V2.65625H2.83398V14.6562H10.0527ZM4.05273 3.875H8.83398L4.05273 9.875V3.875Z" fill="black" fillOpacity="0.87" />
+					</svg>
+
+					{ __( 'Live view of your site', 'amp' ) }
+				</p>
+				<Phone>
+					<iframe
+						className="done__preview-iframe"
+						sandbox="allow-scripts"
```

Comment:
```suggestion
						sandbox="allow-scripts allow-forms allow-popups allow-presentation"
```

Reply:
You got this in just ahead of my comment below. I'll try it in a little while.